lightning-invoice/src/payment.rs

```diff
@@ -773,25 +740,48 @@ where
 	}
 }
 
+/// Data structure used to track information about inflight HTLCs.
+pub struct InFlightHtlcs {
+	/// A map with liquidity value (in msat) keyed by a short channel id and the direction the HTLC
+	/// is traveling in. The direction boolean is determined by checking if the HTLC source's public
+	/// key is less than its destination. See [`InflightHtlcs::used_liquidity_msat`] for more
+	/// details.
+	pub map: HashMap<(u64, bool), u64>
+}
+
+impl InFlightHtlcs {
+	/// Creates a new instance with an optional map of liquidity information.
+	pub fn new(map: Option<HashMap<(u64, bool), u64>>) -> Self {
+		InFlightHtlcs { map: map.unwrap_or(HashMap::new()) }
+	}
+
+	/// Returns liquidity in msat given the public key of the HTLC source, destination, and short
+	/// channel id.
+	pub fn used_liquidity_msat(&self, source: &NodeId, destination: &NodeId, channel_scid: u64) -> Option<&u64> {
```

Comment:
Use `target` instead of `destination` for consistent terminology. Likewise throughout the docs.

Reply:
Addressed in c5206b3bf646b61db957ef7dd80b79538114181d